code/modules/reagents/chemistry/recipes/pyrotechnics.dm

```diff
@@ -3,6 +3,7 @@
 	id = "reagent_explosion"
 	var/strengthdiv = 10
 	var/modifier = 0
+	mob_react = FALSE //noreact explosions are fucky as hell. even a power of 1, being the minimal reaction, will nearly insta-crit and will definitely delimb, depending on armor.
```

Comment:
This is bad and you should feel bad.

Reply:
ok, person who mass produces several exploding plants with hypodermic needles